addons/crm/models/crm_team.py

```diff
@@ -43,6 +43,9 @@ class Team(models.Model):
     lead_all_assigned_month_count = fields.Integer(
         string='# Leads/Opps assigned this month', compute='_compute_lead_all_assigned_month_count',
         help="Number of leads and opportunities assigned this last month.")
+    lead_month_assignment_exceeded = fields.Boolean('Exceed monthly lead assignement', compute="_compute_lead_all_assigned_month_count",
```

Comment:
Should we do "lead_all_assigned_month_exceeded" to keep the naming aligned with the other one?